# "rec" is a keyword in ghc 6.12, use "rec'" instead
# spotted by ben
diff -r a521d8da3e5b src/Darcs/Commands/Record.lhs
--- a/src/Darcs/Commands/Record.lhs	Thu Nov 26 22:57:02 2009 +0100
+++ b/src/Darcs/Commands/Record.lhs	Thu Nov 26 23:17:26 2009 +0100
@@ -134,14 +134,14 @@
 record_cmd opts args = do
     check_name_is_not_option opts
     let (logMessage,_, _) = loggers opts
     withRepoLock (testByDefault opts) $- \repository -> do
-    rec <- if null args then return empty_slurpy
+    rec' <- if null args then return empty_slurpy
            else slurp_recorded repository
     files <- sort `fmap` fixSubPaths opts args
     let non_repo_files = if null files && (not $ null args) then args else []
-    existing_files <- filterM (file_exists rec) files
-    non_existent_files <- filterM (fmap not . file_exists rec) files
+    existing_files <- filterM (file_exists rec') files
+    non_existent_files <- filterM (fmap not . file_exists rec') files
     when (not $ null existing_files) $
          logMessage $ "Recording changes in "++unwords (map show existing_files)++":\n"
     when (not $ null non_existent_files) $
          logMessage $ "Non existent files or directories: "++unwords (map show non_existent_files)++"\n"
